src/diffusers/models/resnet.py

```diff
@@ -149,7 +149,6 @@ def _upsample_2d(self, x, weight=None, kernel=None, factor=2, gain=1):
 
             stride = (factor, factor)
             # Determine data dimensions.
-            stride = [1, 1, factor, factor]
```

Comment:
`F.conv_transpose2d` should have a single int or a tuple of 2 integers for `stride`, not 4 integers.